### ISSUE TITLE
Can you add the `posix` property to the `path` module

### ISSUE THREAD

**Issue by CMCDragonkai** [calvinmetcalf/rollup-plugin-node-globals#13]
It should just refer itself.

I need to use the `join` function, however I need to ensure that it unix style path join.

**Comment by calvinmetcalf** [comment]
I don't think we can count on it being necessarily unix style, more generally, if you need it to definitely be unix style, why do you even need to use the path.join function?
